UK/predictors/scripts/00_UK_census_APIs.R

```diff
@@ -42,21 +42,284 @@ get_census_data_england_2011 <- function(geography_type_code) {
+    rename("frp_50_64_fam_w_kids_10_11_two" = "OBS_VALUE") %>% 
+    left_join(., nomis_get_data(id = "NM_854_1", time = "latest", geography = geography_type_code, measures = 20100,
+                                select = c("geography_code", "OBS_VALUE"), AGE_FRPPUK11 = 4, C_DPCFAMUK11 = 11, C_FMTFAMUK11 = 0)) %>%
+    rename("frp_65_plus_fam_w_kids_10_11_two" = "OBS_VALUE") %>%
+    left_join(., nomis_get_data(id = "NM_854_1", time = "latest", geography = geography_type_code, measures = 20100,
+                                select = c("geography_code", "OBS_VALUE"), AGE_FRPPUK11 = 1, C_DPCFAMUK11 = 12, C_FMTFAMUK11 = 0)) %>%
+    rename("frp_u24_fam_w_kids_12_15_two" = "OBS_VALUE") %>% 
+    left_join(., nomis_get_data(id = "NM_854_1", time = "latest", geography = geography_type_code, measures = 20100,
+                                select = c("geography_code", "OBS_VALUE"), AGE_FRPPUK11 = 2, C_DPCFAMUK11 = 12, C_FMTFAMUK11 = 0)) %>%
+    rename("frp_25_49_fam_w_kids_12_15_two" = "OBS_VALUE") %>% 
+    left_join(., nomis_get_data(id = "NM_854_1", time = "latest", geography = geography_type_code, measures = 20100,
+                                select = c("geography_code", "OBS_VALUE"), AGE_FRPPUK11 = 3, C_DPCFAMUK11 = 12, C_FMTFAMUK11 = 0)) %>%
+    rename("frp_50_64_fam_w_kids_12_15_two" = "OBS_VALUE") %>% 
+    left_join(., nomis_get_data(id = "NM_854_1", time = "latest", geography = geography_type_code, measures = 20100,
+                                select = c("geography_code", "OBS_VALUE"), AGE_FRPPUK11 = 4, C_DPCFAMUK11 = 12, C_FMTFAMUK11 = 0)) %>%
+    rename("frp_65_plus_fam_w_kids_12_15_two" = "OBS_VALUE") %>%
+    left_join(., nomis_get_data(id = "NM_854_1", time = "latest", geography = geography_type_code, measures = 20100,
+                                select = c("geography_code", "OBS_VALUE"), AGE_FRPPUK11 = 1, C_DPCFAMUK11 = 13, C_FMTFAMUK11 = 0)) %>%
+    rename("frp_u24_fam_w_kids_16_18_two" = "OBS_VALUE") %>% 
+    left_join(., nomis_get_data(id = "NM_854_1", time = "latest", geography = geography_type_code, measures = 20100,
+                                select = c("geography_code", "OBS_VALUE"), AGE_FRPPUK11 = 2, C_DPCFAMUK11 = 13, C_FMTFAMUK11 = 0)) %>%
+    rename("frp_25_49_fam_w_kids_16_18_two" = "OBS_VALUE") %>% 
+    left_join(., nomis_get_data(id = "NM_854_1", time = "latest", geography = geography_type_code, measures = 20100,
+                                select = c("geography_code", "OBS_VALUE"), AGE_FRPPUK11 = 3, C_DPCFAMUK11 = 13, C_FMTFAMUK11 = 0)) %>%
+    rename("frp_50_64_fam_w_kids_16_18_two" = "OBS_VALUE") %>% 
+    left_join(., nomis_get_data(id = "NM_854_1", time = "latest", geography = geography_type_code, measures = 20100,
+                                select = c("geography_code", "OBS_VALUE"), AGE_FRPPUK11 = 4, C_DPCFAMUK11 = 13, C_FMTFAMUK11 = 0)) %>%
+    rename("frp_65_plus_fam_w_kids_16_18_two" = "OBS_VALUE") %>%
+    
+    # FRP age band count with two dependent children
```

Comment:
this comment is identical to the one for the block above? is this for 3 or more?